From 8054cd148e7a9e3cfa546d60c06b436fb73cf803 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@ubuntu.com>
Date: Mon, 13 Jan 2014 12:13:27 +0000
Subject: Cope with Kubuntu setting GRUB_DISTRIBUTOR

This is not a very good approach, and certainly not sanely upstreamable;
we probably need to split GRUB_DISTRIBUTOR into a couple of different
variables.

Bug-Ubuntu: https://bugs.launchpad.net/bugs/1242417
Forwarded: not-needed
Last-Update: 2013-12-25

Patch-Name: install-efi-ubuntu-flavours.patch
---
 util/grub-install.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/util/grub-install.c b/util/grub-install.c
index b0c7c7c37..e5e9e439d 100644
--- a/util/grub-install.c
+++ b/util/grub-install.c
@@ -1115,6 +1115,8 @@ main (int argc, char *argv[])
       */
       char *t;
       efi_distributor = bootloader_id;
+      if (strcmp (efi_distributor, "kubuntu") == 0)
+	efi_distributor = "ubuntu";
       switch (platform)
 	{
 	case GRUB_INSTALL_PLATFORM_I386_EFI:
